data/brands/amenity/fuel.json

```diff
@@ -1079,6 +1089,20 @@
         "name": "Crodux"
       }
     },
+    {
+      "displayName": "Crown",
+      "locationSet": {
+        "include": ["us"]
+      },
+      "matchNames": ["crown"],
+      "tags": {
+        "amenity": "fuel",
+        "brand": "Clark",
```

Comment:
A bigger issue is it shouldn't say "Clark" here, but rather something that matches where the wikidata/wikipedia tags point to, so "Crown Central Petroleum" or "Crown" probably.

Reply:
It seems like Clark and Crown are almost one brand now "Clark / Crown", as the Wikipedia article has a "_merge_" tag, and the [Facebook logo](https://nsi.guide/index.html?t=brands&k=amenity&v=fuel&tt=clark) includes both the Crown and Clark Logo, so maybe in this case both "_Clark_" and "_Crown_" named entries should link to the same Wiki*?

Or, perhaps both should have unique wikidata "brand" entries that are separate from "_Clark Brands_"?

Reply:
Wikipedia does not reflect reality. If these 2 wiki pages merge into one in the future, I can send another PR. Crown is not a national chain so it is very easy to upgrade tags again. However, I think redirection inside mediawiki will cover this well.

You still cannot spot any “Clark” at any Crown stations in MD, even they have already merged since 2013.